### DISCUSSION TITLE
Does someone know good Vits Finetune Notebooks or WebUIs?

### DISCUSSION THREAD

**Discussion by IDiMooo** [idiap/coqui-ai-TTS#176] [Q&A] [answered]
Hello! I want to finetune vits model on my dataset and I wanted to ask if anyone knows good Notebooks or Webuis to do it? Or some good documentation on datasets and code?

**Comment by eginhard** [comment, marked as answer]
https://coqui-tts.readthedocs.io/en/latest/finetuning.html

**Reply by IDiMooo** [reply]
I know about this one that's why I'm asking for something more Vits focused.
